docs/sources/dashboards/use-dashboards/index.md

```diff
@@ -105,13 +105,15 @@ The following table provides example relative ranges:
 | This Year              | `now/Y`     | `now/Y`     |
 | Previous fiscal year   | `now-1y/fy` | `now-1y/fy` |
 
-#### Note about Grafana Alerting
+{{% admonition type="Note" %}}
 
-For Grafana Alerting, we do not support the following syntaxes at this time.
+Grafana Alerting does not support the following syntaxes at this time:
 
 - now+n for future timestamps.
 - now-1n/n for "start of n until end of n" because this is an absolute timestamp.
```

Comment:
@brendamuir - I've added a note in the semi-relative time range section about it not being supported in Alerting, but this note was already here and I just wanted to check with you that it's still valid.